src/renderer/modals/VoteTron/fields/VotesField.js

```diff
@@ -0,0 +1,260 @@
+`;
+
+const InfoContainer = styled(Box).attrs(() => ({
+  vertical: true,
+  ml: 2,
+  flex: 1,
+}))``;
+
+const Title = styled(Box).attrs(() => ({
+  horizontal: true,
+  alignItems: "center",
+}))`
+  font-size: 13px;
+  font-weight: 600;
+  cursor: pointer;
+  color: ${p => p.theme.colors.palette.text.shade100};
+  ${IconContainer} {
+    background-color: rgba(0, 0, 0, 0);
+    color: ${p => p.theme.colors.palette.primary.main};
+    opacity: 0;
+  }
+  &:hover {
+    color: ${p => p.theme.colors.palette.primary.main};
+  }
+  &:hover > ${IconContainer} {
+    opacity: 1;
+  }
+`;
+
+const SubTitle = styled(Box).attrs(() => ({
+  horizontal: true,
+}))`
+  font-size: 11px;
+  font-weight: 500;
+  color: ${p => p.theme.colors.palette.text.shade60};
+`;
+
+const InputBox = styled(Box).attrs(() => ({
+  horizontal: true,
+  alignItems: "center",
+}))`
+  flex-basis: 150px;
+  border: 1px solid transparent;
+  border-color: ${p =>
+    p.active ? p.theme.colors.palette.primary.main : p.theme.colors.palette.divider};
+  border-radius: 4px;
+  height: 32px;
+  padding: 2px 0;
+  transition all 200ms ease-in;
+  ${IconContainer} {
+    opacity: 0;
+    background-color: rgba(0,0,0,0);
+    flex: 0 0 24px;
+  }
+  &:hover > ${IconContainer} {
+    opacity: 1;
+  }
+  &:focus, &:focus-within {
+    color: ${p => p.theme.colors.palette.primary.main};
+    border-color: ${p => p.theme.colors.palette.primary.main};
+  }
+`;
+
+const VoteInput = styled.input.attrs(() => ({
+  type: "text",
+  step: 1,
+  min: 0,
+  pattern: "[0-9]",
+  placeholder: 0,
+}))`
+  flex: 1;
+  text-align: right;
+  border: none;
+  height: 100%;
+  padding: 0 6px;
+  transition all 200ms ease-in;
+`;
+
+type SRRowProps = {
+  sr: SuperRepresentative,
+  rank: number,
+  isSR?: boolean,
+  value?: number,
+  disabled?: boolean,
+  onChange: (vote: number) => void,
+  onExternalLink: (address: string) => void,
+};
+
+const SRRow = ({ sr, rank, isSR, value, disabled, onChange, onExternalLink }: SRRowProps) => (
+  <Row disabled={value && disabled} active={!!value}>
+    <IconContainer isSR={isSR}>{isSR ? <Trophy size={16} /> : <Medal size={16} />}</IconContainer>
+    <InfoContainer>
+      <Title onClick={() => onExternalLink(sr.address)}>
+        <Text>{sr.name}</Text>
+        <IconContainer>
+          <ExternalLink size={16} />
+        </IconContainer>
+      </Title>
+      <SubTitle>
+        <Trans i18nKey="vote.steps.castVotes.rank" values={{ rank }}>
+          <b></b>
+        </Trans>
+        {/** @TODO add estimated yield here */}
+      </SubTitle>
+    </InfoContainer>
+    <Box flex="1" />
+    <InputBox active={!!value}>
+      <IconContainer onClick={() => onChange(0)}>
+        <Cross size={16} />
+      </IconContainer>
+      <VoteInput value={value} disabled={disabled} onChange={e => onChange(e.target.value)} />
+    </InputBox>
+  </Row>
+);
+
+const SR_THRESHOLD = 27;
+
+type Props = {
+  t: TFunction,
```

Comment:
I rather use useTranslation hook inside this component instead of passing as a prop.

Reply:
I don't even use it i'll remove this :+1: